backend/commons/src/main/java/org/sonarsource/sonarlint/core/commons/util/git/GitUtils.java

```diff
@@ -189,6 +192,11 @@ public static SonarLintBlameResult blameFromNativeCommand(Path projectBaseDir, S
     throw new IllegalStateException("There is no native Git available");
   }
 
+  private static String getBlameHistoryWindow(long thresholdDate) {
+    var blameLimit = Instant.ofEpochMilli(thresholdDate);
```

Comment:
Related to what I said earlier, we create Instants that we convert to a timestamp, then back to an Instant

Reply:
Fixed.